plugins/PrivacyManager/tests/System/expected/anonymizeVisitInformation_allSitesAllDates.json

```diff
@@ -10381,11 +10381,11 @@
             "location_country": "in",
             "referer_name": "www.example.com",
             "referer_type": "3",
-            "revenue": "$ 0",
-            "revenue_discount": "$ ",
-            "revenue_shipping": "$ ",
-            "revenue_subtotal": "$ ",
-            "revenue_tax": "$ ",
+            "revenue": "USD 0",
+            "revenue_discount": "USD ",
+            "revenue_shipping": "USD ",
```

Comment:
this is in my opinion a regression, as users would expect to see `$ 25` rather than the wordy `USD 25`. Could we restore the old currency symbols as they were?